bin/schemats.ts

```diff
@@ -18,10 +18,12 @@ let argv: any = yargs
     .alias('c', 'conn')
     .nargs('c', 1)
     .describe('c', 'database connection string')
-    .demand('t')
```

Comment:
I think we need to check if neither table parameter nor schema parameter is provided by the user. In this case we can print a help message letting the user know that at least one of these parameters need to be provided.